targettests/Remote-Sim/qvector_init_from_state.cpp

```diff
@@ -31,6 +31,7 @@ __qpu__ void test_init_large_state() {
 
 __qpu__ void test_state_param(cudaq::state* state) {
   cudaq::qvector q1(state);
+  x(q1);
```

Comment:
Q: Does addition of this operation make the test deterministic?

Reply:
No, this part is just for making the test better - if it changes the output every time it's called we can test that iterations fip the qubit values and not just get stale values.